src/components/modules/contacts/ui/cards/MyContactsCards.tsx

```
@@ -0,0 +1,135 @@
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  const [selectedContact, setSelectedContact] = useState<Contact | null>(null);
+
+  const totalPages = Math.ceil(contacts.length / itemsPerPage);
+  const paginatedContacts = contacts.slice(
+    (currentPage - 1) * itemsPerPage,
+    currentPage * itemsPerPage,
+  );
+
+  return (
+    <>
+      {false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)